### ISSUE TITLE
Caching is unnecessary

### ISSUE THREAD

**Issue by Chronial** [selwin/django-user_agents#18]
Caching the results seems unnecessary https://github.com/selwin/django-user_agents/blob/master/django_user_agents/utils.py#L41, since `uap-python` doesn't actually load the yamls anymore, but they are converted to python in the build process.

See here https://github.com/ua-parser/uap-python/blob/master/setup.py#L20 and here https://github.com/ua-parser/uap-python/blob/master/ua_parser/user_agent_parser.py#L548.

So this is loaded and imported once and then cached by the python importing system anyways.

**Comment by acu192** [comment]
I agree. In fact, caching will likely be slower than simply re-parsing the UA on each request. A warmed-up parser takes microseconds to parse the UA. See below:

<img width="455" alt="Screen Shot 2020-02-24 at 1 01 59 PM" src="https://user-images.githubusercontent.com/9354539/75182395-ea904d80-5705-11ea-82a5-1096dbef17c7.png">

<img width="620" alt="Screen Shot 2020-02-24 at 12 53 23 PM" src="https://user-images.githubusercontent.com/9354539/75182399-ecf2a780-5705-11ea-9e12-8533a93ca9be.png">

No reason to put this extra burden on your cache server when it doesn't help you at all.

**Comment by boatcoder** [comment]
I'm also seeing some issues with redis parsing and ua-parser==0.15.0 as well.  Haven't managed to pin it down yet though.
